Jenkinsfile

```diff
@@ -11,9 +13,13 @@ pipeline {
       }
 
       steps {
-        withCredentials([string(credentialsId: 'npm-bringg', variable: 'NPM_TOKEN')]) { writeFile file: '.npmrc', text: "//registry.npmjs.org/:_authToken=${env.NPM_TOKEN}" }
+        // package.json should not have private dependencies
+        // we should be able to install dependencies without credentials
         sh 'yarn'
```

Comment:
we are not using yarn at bringg, can you change it to be npm?

Reply:
changed